sklearn/metrics/scorer.py

```diff
@@ -187,15 +187,67 @@ def _factory_args(self):
 
 
 def get_scorer(scoring):
+    """Converts an estimator scoring strategy into a valid callable
```

Comment:
https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings
"The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line."